App/AuthProvider.js

```diff
@@ -67,8 +71,12 @@ class AuthProvider {
         try {
             return await this.clientApplication.acquireTokenSilent(tokenRequest);
         } catch (error) {
-            console.log('Silent token acquisition failed, acquiring token interactive');
-            return await this.getTokenInteractive(tokenRequest);
+            if (error instanceof InteractionRequiredAuthError) {
+                console.log('Silent token acquisition failed, acquiring token interactive');
+                return await this.getTokenInteractive(tokenRequest);
+            }
+
+            console.log(error);
```

Comment:
Update the failureTemplate to errorTemplate, check [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/src/request/InteractiveRequest.ts).